.travis.yml

```diff
@@ -1,6 +1,12 @@
 language: java
+python:
+  - "2.7"
 jdk:
   - oraclejdk8
```

Comment:
I don't know what this line means, but I remind you that Truffle has to build with JDK7.